doc/queries.md

```diff
@@ -1,16 +1,16 @@
 # Searching bugs
 
-You can search bugs using a micro query language for both filtering and sorting. A query could looks like this:
+You can search bugs using a micro query language for both filtering and sorting. A query could look like this:
 
 ```
 status:open sort:edit
 ```
 
 A few tips:
 
-- query are case insensitive.
-- you can combine as much qualifier as you want.
-- you can use quotation for multi-word search term. For example, use `author:"René Descartes"` to search with a complete name.
+- queries are case insensitive.
+- you can combine as many qualifiers as you want.
+- you can use double quotes for multi-word search terms. For example, `author:"René Descartes"` searches for bugs opened by René Descartes, whereas `author:René Descartes` looks for issues opened by anyone named René containing “Descartes” in their content.
```

Comment:
Does the search normalize diacritical marks (i.e. searching for “rene” would still find issues opened by someone named “René”)?

Reply:
It should, I'm not sure it is already working though. I will try to make it work.

Reply:
`author:` will only match on the person that opened the bug. Comments are not indexed at the moment, it will come at a later stage.

Reply:
Does https://github.com/MichaelMure/git-bug/pull/41/commits/fc0f058a00eb190d79fcaff8b63dc1f669b89d45 make this better?

Reply:
It will be correct eventually, but text search is not implemented at the moment because it would require a clever full-text indexing. `author:"René Descartes"` will return an error.